### PR TITLE
DROOLS-5067: Add Bound Fact Property checkmark

### PR DESCRIPTION
**Pull request by jomarko**
For more details see: https://issues.redhat.com/browse/DROOLS-5067

### CODE REVIEW

**Comment by yesamer** [comment]
jenkins retest this please

**Comment by dupliaka** [comment]
Thanks @jomarko , please do not forget to fix it also for /expression see: https://issues.redhat.com/secure/attachment/12465472/ExpressionTick.webm

**Comment by yesamer** [comment]
@jomarko @dupliaka  I'm not totally convinced regarding this fix. I mean, when I implemented that tick, its scope was: the user **manually selected** that instance / property, which is different than highlight the instance / property in scope. With this change, the tick and the blue highlight will have the same meaning, and probably this is not the scope of that tick.

**Comment by jomarko** [comment]
@yesamer Ok, I see your point, I was not part of UX planning in the original jira, however still think this change makes more consistent behavior. For sure just my opinion, I propose to ask on weekly meeting where is broader audience. What do you think?

**Comment by jomarko** [comment]
@dupliaka ok, will incorporate also for expressions according to results of the discussion about this approach.

**Comment by yesamer** [comment]
> @yesamer Ok, I see your point, I was not part of UX planning in the original jira, however still think this change makes more consistent behavior. For sure just my opinion, I propose to ask on weekly meeting where is broader audience. What do you think?

@jomarko It sounds good :)

**Comment by jomarko** [comment]
@dupliaka your requirement from https://github.com/kiegroup/drools-wb/pull/1314#issuecomment-586151971 should be incorporated now.

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java [hunk 1]
```diff
@@ -71,9 +71,9 @@ public void showAll() {
 
     @Override
     public void selectProperty(String factName, List<String> propertyParts) {
-        final ListGroupItemView listGroupItemView = listGroupItemViewMap.get(factName);
-        if (!listGroupItemView.isShown()) {
-            onToggleRowExpansion(listGroupItemView, false);
+        final ListGroupItemView instanceListGroupItemView = listGroupItemViewMap.get(factName);
+        if (!instanceListGroupItemView.isShown()) {
```
Comment (yesamer) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java]: @jomarko There is a particular reason about this rename?

**Comment by yesamer** [comment]
@jomarko We have 2 failing tests, can you please check them?

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenterTest.java [hunk 2]
```diff
@@ -54,8 +56,8 @@
     @Mock
     private DivElement divElementMock;
 
-    @Mock
-    private FieldItemPresenter fieldItemPresenterMock;
+    @Spy
+    private FieldItemPresenter fieldItemPresenterSpy;
```
Comment (yesamer) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenterTest.java]: @jomarko `@Spy` must be initialized, this is the reason of failing tests. (NPE is thown)

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java (hunk 1 above)
Comment (jomarko) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java]: From my point of view `listGroupItemView` just says what the variable is instance off, while `instanceListGroupItemView` tries to provide more information, it is an `ListGroupItemView` representing an `Instance` , however if you think my rename is inapproprtiate I can revert it. Or if you have different proposal for name.

Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java]: @jomarko Thank you, fine for me.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenterTest.java (hunk 2 above)
Comment (jomarko) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenterTest.java]: Done

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
@dupliaka just friendly reminder, if you have a moment to review.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
@dupliaka done, please have a look

**Comment by jomarko** [comment]
@yesamer the code has changed slightly since your last review, not sure if you want to re-review or we can merge.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by yesamer** [comment]
@jomarko You can proceed.

**Comment by jomarko** [comment]
@yesamer thank you, once green will ask for merge.

**Comment by jomarko** [comment]
@kiegroup/gatekeepers could we please merge?

**Comment by manstis** [comment]
@jomarko SonarCloud is reporting zero coverage.. but I see you have written tests; so merging.

**Comment by yesamer** [comment]
@manstis Unfortunetely test coverage is currently broken :/

**Pull request merged**